src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,13 @@
 
 -->
 ## Upcoming Release
+* Added mandatory boolean parameter `-forceUpdateOSDiskForEphemeral` to the `Set-AzVmssVM` cmdlet to enhance OS disk management for ephemeral VMs.
```

Comment:
@haagha why is it a mandatory parameter? It should be optional and set to false by default

Reply:
@viveksingla08 Thanks for point this out. I will correct this.